# Statistics Percentiles

### Find 10% percentile for Max_Pulse

#### Code

In [1]:
import pandas as pd
import numpy as np

full_health_data = pd.read_csv("data.csv", header=0, sep=",")

Max_Pulse= full_health_data["Max_Pulse"]
percentile10 = np.percentile(Max_Pulse, 10)

print(percentile10)


120.0


#### Explanation

- `Max_Pulse = full_health_data["Max_Pulse"]` isolates the variable `Max_Pulse` from the full health data set.
- `np.percentile()` is used to define that we want the 10% percentile from `Max_Pulse`.
- The 10% percentile of `Max_Pulse` is 120. That means that 10% of all the training sessions have a `Max_Pulse` of 120 or lower.